components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -806,7 +806,7 @@ void RRTMGPRadiation::run_impl (const double dt) {
   const auto do_aerosol_rad = m_do_aerosol_rad;
 
   // Are we going to update fluxes and heating this step?
-  auto ts = timestamp();
+  auto ts = end_of_step_ts();
```

Comment:
@brhillman I changed this, since it seemed wrong that RRTMGP used the timestamp at the beginning of this step down below, where orb params are used. However, with this mod, we also change _when_ rad runs. E.g., with rad freq 2, rad would run
- before this change: on steps 1, 3, 5, ..
- after this change: on steps 2, 4, 6, ...

Not sure if this is ok.